docs/implementation.html

```diff
@@ -350,12 +350,28 @@
 /consumers/[group_id]/owners/[topic]/[partition_id] --> consumer_node_id (ephemeral node)
 </pre>
 
+<h4><a id="impl_clusterid" href="#impl_clusterid">Cluster Id</a></h4>
+
+<p>
+    The cluster id is an unique and immutable identifier assigned to a Kafka cluster. The cluster id is 22 characters long and the allowed characters is defined by the regular expression [a-zA-Z0-9_\-]+, which corresponds to the characters used by the URL-safe Base64 variant. It is auto-generated when the cluster is started for the first time.
+</p>
+
+<pre>
+/cluster/id  --> {"version":...,"id": ...} (persistent node)
```

Comment:
I suggest we don't include this here and instead include it in the following page:

https://cwiki.apache.org/confluence/display/KAFKA/Kafka+data+structures+in+Zookeeper

It's just easier to remember to update if all of the ZK structures are in the same place.